app/vep/vep_resources.py

```diff
@@ -54,12 +61,30 @@ async def submit_vep(request: Request):
         else:
             raise Exception
     except MaxBodySizeException:
-        return HTTPException(
-            status_code=status.HTTP_413_REQUEST_ENTITY_TOO_LARGE,
-            detail="Maximum file size limit exceeded.",
-        )
+        return response_error_handler(result={"status": 413})
     except Exception as e:
-        return HTTPException(
-            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
-            detail="Could not submit VEP job/workflow.",
-        )
+        return response_error_handler(result={"status": 500})
+
+
+@router.get("/submissions/{job_id}/status", name="submission_status")
+async def submit_vep(request: Request, job_id: str):
+    try:
+        workflow_status = await get_workflow_status(job_id)
+        return JSONResponse(content=workflow_status.dict())
```

Comment:
I may be misreading the python here; but I don't think the shape of the response conforms to the spec.

Reply:
What do you mean?
This response is not in the spec?
`{
    "job_id": "5SYhtjFhDfmzx2",
    "status": "FAILED"
}`

Reply:
The spec calls the field `submission_id` 😉

Reply:
Ah, I got confused by having two names for submission_id, in the url we call it job_id and in the response we call it submission_id! we should update one of them!

Reply:
Done